src/components/AddressResolver/EndpointEntry/EndpointEntry.tsx

```diff
@@ -45,6 +45,8 @@ export const EndpointEntry: FunctionComponent<EndpointEntryProps> = ({
   const [endpointName, setEndpointName] = useState(name);
   const [endpointApiHeader, setEndpointApiHeader] = useState(apiHeader);
   const [endpointApiKey, setEndpointApiKey] = useState(apiKey);
+  const [hoverState, setHoverState] = useState(false);
```

Comment:
use tw `group-hover`, less logic and might be more convenient

Reply:
done.